docs/configuration/configuration-file.mdx

```diff
@@ -276,25 +502,12 @@ resolve:
         envVariable: SECRET_AUTH
 ```
 
-The first match takes precedence when a URL matches multiple patterns. Therefore, only the header from the first match will be used in the request.
-
-
-### features.openapi
-
-The `features.openapi` section configures features (availability and options) and theming (style, fonts, colors) for API reference documentation generated from API definitions.
-
-There are two products: Redoc Community Edition (CE) and Redocly Reference. Reference supports all configuration options from Redoc Community Edition plus a set of premium options.
-
-Find the full list of supported options on the [Reference docs configuration page](../../api-reference-docs/configuration/index.mdx).
-
-### registry
+If the URL matches multiple patterns, the first match takes precedence. Therefore, only the header from the first match will be used in the request.
 
-The `registry` section is reserved for future API registry features.
+## features.openapi
 
-Read more about the [`registry` section](registry.md).
+The `features.openapi` section is where you configure how your published reference docs will look (style, fonts, colors — known as _theming_) and behave (enable/disable search, show/hide API reference docs download option  — known as _functionality_).
 
-## Schema reference
+There are two products impacted by this section: Redoc Community Edition (CE) and Redocly Reference. Reference supports all configuration options from Redoc Community Edition plus a set of premium options.
```

Comment:
issue: The developer portal is also affected when someone uses the API catalog feature.